src/modules/AnalysisExecution/components/SubmissionCodeViewer/ResultFile/components/Summary/Incidence/presenter.jsx

```diff
@@ -73,12 +96,17 @@ export default function SummaryIncidence({ resultInfo = [], className }) {
           </div>}
           field={'timeAtRisk'}
         />
-        <TableCellText
+        <DynamicCell
           header={<div {...classes('header')}>
-            <span>Rate [+|-]</span>
-            <span {...classes('header-subtitle')}>per 1k years</span>
+            <span>Rate [
+              <span {...classes('span-active')} onClick={() => changeRatio(10)}> + </span>
```

Comment:
Is it always x10 or /10? Or is it currently possible to change ratio in arbitrary way?

Reply:
It is always x10 or /10

Reply:
then, just storing decimal exponent value and doing +1/-1 on it would be much more concise and would not require divisions.

Reply:
agree, fixed